eng/common/scripts/Update-DocsMsPackages.ps1

```diff
@@ -18,10 +18,17 @@ packages which have not released to a central package manager.
 .PARAMETER DocRepoLocation
 Location of the docs.microsoft.com reference docs repo.
 
+.PARAMETER PackageSourceOverride
+Optional parameter to supply a different package source (useful for daily dev
+docs generation from pacakges which are not published to the default feed)
+
 #>
 param (
   [Parameter(Mandatory = $true)]
-  $DocRepoLocation # the location of the cloned doc repo
+  [string] $DocRepoLocation, # the location of the cloned doc repo
+
+  [Parameter(Mandatory = $false)]
+  [string] $PackageSourceOverride
```

Comment:
Where do you plan to consume this?

Reply:
It's used in the domain-specific business logic: https://github.com/Azure/azure-sdk-for-net/pull/24199/files/59f0789a88c769ce1044e88f4ac5108998e59802#diff-1d28dc86baa237ca6efe6224431142a9e8bbff66fa10a2c20c69031a22f5b387R376

Reply:
It might be worth adding a comment for where this is consumed so that we don't accidently remove this parameter thinking that it isn't used.